functional_tests/scylla_operator/test_functional.py

```diff
@@ -154,6 +155,44 @@ def test_rolling_restart_cluster(db_cluster):
             f"iotune was run after reboot on {pod_name_and_status['name']}"
 
 
+@pytest.mark.required_operator("v1.10.0")
+def test_add_new_node_and_check_old_nodes_are_cleaned_up(db_cluster):
+    log_followers = {}
+    for node in db_cluster.nodes:
+        for keyspace in db_cluster.nodes[0].run_cqlsh('describe keyspaces').stdout.split():
+            log_followers[f"{node.name}--{keyspace}"] = node.follow_system_log(patterns=[
+                f"api - force_keyspace_cleanup: keyspace={keyspace} "])
+
+    def wait_for_cleanup_logs(log_follower_name, log_follower, db_cluster):
+        db_rf = len(db_cluster.nodes)
+        while not list(log_follower):
+            # NOTE: log lines located as last may not be caught fast enough because of the chunk-size limitation.
```

Comment:
hmm, can we `echo "some log message" >> <path to scylla log> to achieve it?

Reply:
It is done here by creating and dropping a keyspace.

Reply:
yes, I see it, but was wondering if we could have done it in a way that does not change db schema...